src/Services/MiningRewardsService.ts

```diff
@@ -0,0 +1,22 @@
+import sequelize from "./Database";
+import {MiningReward} from "../Types/MiningRewardType";
+import config from "../Config/Config";
+
+export class MiningRewardsService {
+
+    public async storeMiningRewards(miningRewards: Array<MiningReward>) {
+        return await sequelize.sequelize.query(
```

Comment:
Nope. You should use the array with `bulkCreate`.

Should be: `MiningRewards.bulkCreate(miningRewards)`